pages/frauen/index.vue

```diff
@@ -103,3 +107,19 @@ export default {
   },
 }
 </script>
+
+<style>
```

Comment:
i think we should use scoped everywhere where we do not want the css applied to all other components/pages